src/libhirte/common/cfg.c

```diff
@@ -137,6 +141,29 @@ int cfg_load_from_file(struct config *config, const char *config_file) {
         return ini_parse(config_file, parsing_handler, config);
 }
 
+char *is_file_name_ending_with_conf(char *file_name, char *conf_file_enfing) {
+        return strstr(file_name, conf_file_enfing);
+}
+
+int cfg_load_from_dir(struct config *config, const char *custom_config_directory) {
+        struct dirent **all_files_in_dir = NULL;
+        int number_of_files = 0;
+        char *conf_file_enfing = ".conf";
+
+        number_of_files = scandir(custom_config_directory, &all_files_in_dir, NULL, alphasort);
+        if (number_of_files == -1) {
+                return ENOENT;
+        }
+        for (int i = 1; i <= number_of_files; i++) {
+                char *file_name = all_files_in_dir[i]->d_name;
+                char *files_ends_with_conf = is_file_name_ending_with_conf(file_name, conf_file_enfing);
```

Comment:
If a function has the prefix `is_` in its name, I'd expect it to return a `bool`. In this case, the resulting code would be a bit more concise as well:
```C
if (is_file_name_ending_with_conf(file_name, conf_file_ending)){
   cfg_load_from_file(config, file_name);
}
```
Could you change the `is_file_name_ending_with_conf` to return a bool?

Reply:
Now it should be int so I could use it in scandir